docs/wallet-web-api.md

```diff
@@ -0,0 +1,1518 @@
+
+- Supported content types are:
+
+    - `application/json`
+
+- 
+
+```javascript
+{
+    "Left": "Sample error"
+}
+```
+
+- 
+
+```javascript
+{
+    "Right": []
+}
+```
+
+## GET /api/wallets
+
+#### Authentication
+
+
+
+Clients must supply the following data
+
+
+#### Response:
+
+- Status code 200
+- Headers: []
+
+- Supported content types are:
+
+    - `application/json`
+
+- 
+
+```javascript
+{
+    "Left": "Sample error"
+}
+```
+
+- 
+
+```javascript
+{
+    "Right": []
+}
+```
+
+- 
+
+```javascript
+{
+    "Right": [
+        {
+            "cwAmount": {
+                "getCoin": 0
+            },
+            "cwAddress": "1fSCHaQhy6L7Rfjn9xR2Y5H7ZKkzKLMXKYLyZvwWVffQwkQ",
+            "cwMeta": {
+                "cwType": "CWTPersonal",
+                "cwCurrency": "ADA",
+                "cwName": "Personal Wallet"
+            }
+        }
+    ]
+}
+```
+
+- 
+
+```javascript
+{
+    "Right": [
+        {
+            "cwAmount": {
+                "getCoin": 0
+            },
+            "cwAddress": "1fSCHaQhy6L7Rfjn9xR2Y5H7ZKkzKLMXKYLyZvwWVffQwkQ",
+            "cwMeta": {
+                "cwType": "CWTPersonal",
+                "cwCurrency": "ADA",
+                "cwName": "Personal Wallet"
+            }
+        },
+        {
+            "cwAmount": {
+                "getCoin": 0
+            },
+            "cwAddress": "1fSCHaQhy6L7Rfjn9xR2Y5H7ZKkzKLMXKYLyZvwWVffQwkQ",
+            "cwMeta": {
+                "cwType": "CWTPersonal",
+                "cwCurrency": "ADA",
+                "cwName": "Personal Wallet"
+            }
+        }
+    ]
+}
+```
+
+- 
+
+```javascript
+{
+    "Right": [
+        {
+            "cwAmount": {
+                "getCoin": 0
+            },
+            "cwAddress": "1fSCHaQhy6L7Rfjn9xR2Y5H7ZKkzKLMXKYLyZvwWVffQwkQ",
+            "cwMeta": {
+                "cwType": "CWTPersonal",
+                "cwCurrency": "ADA",
+                "cwName": "Personal Wallet"
+            }
+        },
+        {
+            "cwAmount": {
+                "getCoin": 0
+            },
+            "cwAddress": "1fSCHaQhy6L7Rfjn9xR2Y5H7ZKkzKLMXKYLyZvwWVffQwkQ",
+            "cwMeta": {
+                "cwType": "CWTPersonal",
+                "cwCurrency": "ADA",
+                "cwName": "Personal Wallet"
+            }
+        },
+        {
+            "cwAmount": {
+                "getCoin": 0
+            },
+            "cwAddress": "1fSCHaQhy6L7Rfjn9xR2Y5H7ZKkzKLMXKYLyZvwWVffQwkQ",
+            "cwMeta": {
+                "cwType": "CWTPersonal",
+                "cwCurrency": "ADA",
+                "cwName": "Personal Wallet"
+            }
+        }
+    ]
+}
+```
+
+## POST /api/wallets
+
+#### Authentication
+
+
+
+Clients must supply the following data
+
+
+#### Request:
+
+- Supported content types are:
+
+    - `application/json`
+
+- Example: `application/json`
+
+```javascript
+{
+    "cwBackupPhrase": {
+        "bpToList": [
+            "me",
```

Comment:
It was unclear to me that this was just a placeholder. Can you pick 12 random words from the dictionary we use to sample backup words from instead?